src/seedsigner/gui/screens/screen.py

```diff
@@ -447,6 +450,7 @@ def _run(self):
         while True:
             ret = self._run_callback()
             if ret is not None:
+                print("Exiting ButtonListScreen due to _run_callback")
```

Comment:
If it makes sense to keep this logging output (which seems fine to me), then screen.py should probably import logger and do `logger.info("Exiting ButtonListScreen due to _run_callback")`

Reply:
Done!